### PR TITLE
Error messages

### LINKED ISSUE #95: Add UI error/loading feedbacks
Worth move to cross-platform UI components (https://github.com/tasitlabs/tasit/issues/88) before that?

### PR DESCRIPTION
**Pull request by marcelomorgado**
### Issue link

#95 

### Auto-close the issue?

Closes #95 

### Types of changes

New feature (non-breaking change that adds functionality)

Technical debt (a code change that doesn't fix a bug or add a feature but makes something clearer for devs)

### Notes

Waiting for a response to the question: https://github.com/tasitlabs/tasit/issues/95#issuecomment-471680764

### CODE REVIEW

**Comment by pcowgill** [comment]
> Waiting for a response to the question: #95 (comment)

Just replied!

File: demo/App.js [hunk 1]
```diff
@@ -2,11 +2,13 @@ import React from "react";
...
 import AppNavigator from "./AppNavigator";
-import { Action } from "tasit-sdk";
-import tasitSdkConfig from "./config/default.js";
 import { Provider } from "react-redux";
 import { createStore } from "redux";
 import demoApp from "./redux/reducers";
+import { Action } from "tasit-sdk";
+const { ConfigLoader } = Action;
+import tasitSdkConfig from "./config/default.js";
+import { checkBlockchain } from "./helpers.js";
```
Comment (pcowgill) [inline demo/App.js]: We can do without the .js suffix if we like

File: demo/screens/ListLandForSaleScreen.js [hunk 2]
```diff
@@ -36,6 +40,8 @@ export class ListLandForSaleScreen extends React.Component {
       orders.push(order);
     }
 
+    if (orders.length == 0) console.warn("There are no land for sale.");
```
Comment (pcowgill) [inline demo/screens/ListLandForSaleScreen.js]:
```suggestion
    if (orders.length == 0) console.warn("There isn't any land for sale right now.");
```

File: demo/App.js (hunk 1 above)
Comment (marcelomorgado) [reply demo/App.js]: Yes, sure.

Comment (marcelomorgado) [reply demo/App.js]: Done.

Comment (pcowgill) [reply demo/App.js]: Thanks. Resolving.

**Pull request merged**